proxy/src/main/java/com/wavefront/agent/AbstractAgent.java

```diff
@@ -726,6 +726,16 @@
   @Parameter(description = "")
   protected List<String> unparsed_params;
 
+  @Parameter(names = {"--reporterDelayMillis"}, description = "Delay time for delta counter reporter." +
+          "Defaults to 60000")
+  protected  long reportInterval = 3000;
```

Comment:
this is not good naming scheme
you should have time unit mentioned in the variable name
i think you should rename this to 
reportIntervalSeconds = 1